src/MiniCover/MiniCover.csproj

```diff
@@ -30,7 +30,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\MiniCover.HitServices\MiniCover.HitServices.csproj" />
+    <ProjectReference Include="..\MiniCover.HitServices\MiniCover.HitServices.csproj"
+      PrivateAssets="All" />
```

Comment:
Flag with `PrivateAssets=All` to don't include MiniCover.HitServices as nuget dependency.